src/components/Logo/AssetLogo.tsx

```diff
@@ -77,11 +82,22 @@ export default function AssetLogo({
 
   const [src, nextSrc] = useTokenLogoSource(address, chainId, isNative, backupImg)
   const L2Icon = getChainInfo(chainId)?.circleLogoUrl
+  const [imgLoaded, setImgLoaded] = useState(false)
```

Comment:
nit: I'd prefer this state match the props to which it is passed. Could you rename `imgLoading` and `useBg` to both be `imgLoaded` so there is less renaming of this boolean?